client/src/main/java/com/paypal/selion/platform/grid/MobileNodeType.java

```diff
@@ -55,6 +57,12 @@ public static String getSupportedMobileNodesAsCSV() {
      * @return - A {@link MobileNodeType} enum that represents a SeLion compliant mobile node type.
      */
     public static MobileNodeType getMobileNodeType(String mobileNodeType) {
+        // this is a mandatory field ??? should we default it to appium if they haven't specified since appium supports
+        // both ios and android
+        if (StringUtils.isEmpty(mobileNodeType)) {
+            
```

Comment:
Why this empty `if` statement... We can address the mandatory-ness of `mobileNodeType` with a different change.